### ISSUE TITLE
Tailscale SSH on browser closes immediately

### ISSUE THREAD

**Issue by Oujiii** [tailscale/tailscale#6360]
### What is the issue?

I tried activating Tailscale SSH on several nodes. I am able to activate it, but when I try rendering the terminal from the browser, it closes and gives the message "Tailscale SSH encountered an error during the session, please try again or contact support."

Tried in several nodes and the ACLs are working just fine (because when they are not, the SSH button doesn't show up for the specific node).

### Steps to reproduce

Just running tailscale up --ssh on any node

### Are there any recent changes that introduced the issue?

Not at all

### OS

Linux

### OS version

Debian 10

### Tailscale version

1.32.2

### Bug report

BUG-b911571e50f92be8966175968437ddd269b1a4151e3c3be03c9ceeaf71703d86-20221116213558Z-1a83c3cc6ead2ac0

**Comment by bradfitz** [comment]
>  but when I try rendering the terminal from the browser

Which browser?

**Comment by Oujiii** [comment]
Chrome.

**Comment by mihaip** [comment]
Which username are you trying to connect as? Does doing an `ssh user@host` from the command line of (a machine that's running Tailscale) work?

**Comment by Oujiii** [comment]
Before your question, some ACLs were wrong (because my node was tagged), so I was unable to ssh from the terminal using the tailscale IP (tailscale would deny it). I resolved these ACLs issues (now I can ssh from that device to the one I'm testing), but I'm still seeing the same error.

**Comment by mihaip** [comment]
It looks like your ACLs are still somewhat restrictive. The SSH policy allows users (`autogroup:members`) to connect, but the lower-level ACLs for all connections is expressed only in terms of tags, and the ephemeral node in your browser has no tags (it's associated with your user). Your options are:

1. Relax the ACLs to also allow individual users (or groups) to connect to servers
2. Add the desired ACLs to the ephemeral node for the SSH console (you can do this via the admin panel by finding the `tailscale-ssh-console-...` item in the machine tab and choosing "Edit ACL Tags")
![image](https://user-images.githubusercontent.com/513813/202584290-45a6b62d-0bc3-482a-b95d-ccf8b07aa877.png)

**Comment by Oujiii** [comment]
Oh, I tested tagging the ephemeral node and it worked. Also tested just adding my user to the ACLs and it is working now. It's a shame that I'll have to go this extra step, but it is what it is. Since a tag is required, would be nice if we could "tag" automatically those ephemeral nodes for the purpose of using SSH from the browser or maybe just a specific name to put on `src` that we could have for those wanting to SSH from the browser, like `ssh-browser` or something, this could be on the documentation 

If anyone else is reading this, my ACLs right now looks like this for SSH:

```
{
			"action": "accept",
			"src":    ["autogroup:members", "tag:workstation-remote"],
			"dst":    ["tag:servers-local"],
			"users":  ["autogroup:nonroot", "root"],
		}
```
This node is currently a part of `tag:servers-local` which HAD those specific ACLs up until now:

```
{
			"action": "accept",
			"src":    ["tag:workstation-remote"],
			"dst":    ["tag:servers-local:*", "net-local:*"],
		}
```

What I did (and will have to do for all the rules that I want to allow SSH from the browser is adding `autogroup:members` or `myuser@microsoft.com` to the `src` of my network ACL. I tested with both and they both work, so I'm leaving this here if anyone faces a similar issue.

Thank you @mihaip and the whole Tailscale team for the amazing help again!
